### DISCUSSION TITLE
first-class subtask

### DISCUSSION THREAD

**Discussion by azzamsa** [BaldissaraMatheus/Tasks.md#128] [Ideas]
Hi,

Is there any chance we have a first-class subtask like Task.org.
The first-class subtask is a complete stand-alone task linked to the parent, not just a checkbox.
So the child/sub-task will also have all the properties the parent had.

Thanks!

**Comment by BaldissaraMatheus** [comment]
I'm not familiar with Tasks.org, but I will take a look. That's also something I was considering to add, I'm not really sure yet on how I will implement that, but I'll experiment a bit for next releases.

I think it's similar to #104
